site/content/en/docs/commands/start.md

```diff
@@ -36,6 +36,7 @@ minikube start [flags]
       --disk-size string                  Disk size allocated to the minikube VM (format: <number>[<unit>], where unit = b, k, m or g). (default "20000mb")
       --dns-domain string                 The cluster dns domain name used in the kubernetes cluster (default "cluster.local")
       --dns-proxy                         Enable proxy for NAT DNS requests (virtualbox driver only)
+      --docker-base-image string          The base image to use for docker/podman drivers. Intended for local development. (default "gcr.io/k8s-minikube/kicbase:v0.0.9@sha256:82a826cc03c3e59ead5969b8020ca138de98f366c1907293df91fc57205dbb53")
```

Comment:
does this accept values without sha ? because local images dont have repoTag Sha
(I think it will be fine but would be good if u confirm it works for passing without sha)

Reply:
as stated in the PR description, this works without sha